internal/command/default.provisioners.yaml

```diff
@@ -216,6 +216,96 @@
     - username
     - password
 
+
+- uri: template://default-provisioners/postgres
+  # By default, match all redis types regardless of class and id. If you want to override this, create another
+  # provisioner definition with a higher priority.
+  type: postgres-instance
+  description: Provisions a dedicated database on a shared postgres instance
```

Comment:
Descriptions  mentions a dedicated database on a shared instance. This provisioner is for just an instance so description should be updated.